packages/driver/src/cypress/error_messages.ts

```diff
@@ -1237,6 +1237,22 @@ export default {
         docsUrl: 'https://on.cypress.io/session-api',
       },
     },
+    cannot_visit_previous_origin (args) {
+      return {
+        message: stripIndent`\
+          ${cmd('visit')} failed because you are attempting to visit a URL from a previous origin inside of ${cmd('origin')}.
+
+          Instead of placing the ${cmd('visit')} inside of ${cmd('origin')}, the ${cmd('visit')} should be placed at the top-level.
```

Comment:
Maybe?

```suggestion
          Instead of placing the ${cmd('visit')} inside of ${cmd('origin')}, the ${cmd('visit')} should be placed outside of the ${cmd('origin')} block.
```

Reply:
Updated [b43b179](https://github.com/cypress-io/cypress/pull/21144/commits/b43b17997d9b4e22ddae680827ce11a333a2bc61)